README.md

```diff
@@ -12,7 +29,22 @@ git clone https://github.com/JeeveshN/Retrieve-Wifi-Passwords.git or download th
 python pass.py
 ```
 ![image](https://media.giphy.com/media/hoJvidbWg5vUs/giphy.gif)
-### Retrieve Based on Wifi Name
+### Retrieve in JSON format
+```
+python pass.py --json
+```
+Example of output:
+```json
+{
+  "Example-Wifi-SSID" : "WifiPassword1234",
+  "WifiSSID2" : "p4$$w0rD"
+}
+
+```
+
+
+### Retrieve Based on Wifi Name [REMOVED] 
+Removed because it is not useful at all.
```

Comment:
I would like you to keep this functionality

Reply:
Okay